Core1Platform/01_General.md

```diff
@@ -0,0 +1,17 @@
+# General information
+
+The *Actindo Core1 Platform* offers a comprehensive solution for efficient digital Unified Commerce. It is an API-first and cloud-native *Digital Operations Platform (DOP)* that provides you with an easy and fast integration of new business models in an existing IT landscape.   
+The platform brings together a high degree of automation and process integration. It offers a fast time to market as well as tremendous relief and time savings in day-to-day operations.   
+From product data, purchasing, and inventory to customer management, monitoring and reporting, everything is made available centrally in one place. 
+
+The *Actindo Core1 Platform* supports your business processes, regardless of the field of application you need a solution for:
+- You have an existing legacy system in use and want to switch from B2B (business-to-consumer) to D2C (direct-to-consumer).
+- You want to replace an outdated and inflexible IT infrastructure.
+- You are a young start-up which wants a seamless transition to be ready for further growth.   
+  
+The *Actindo Core1 Platform* stands for a loose coupling of modules. This is implemented by the following:   
+- Although a common code base is used, the platform is highly customizable with freely selectable modules and plug-ins, which are maintainable. Any component can be replaced for a true best-of-breed approach, but it can also operate as a standalone solution.   
+- You can individually adapt the dynamic data models to the specific needs of your company. You can easily adjust data streams so that you do not only manage data, but also understood it. The platform provides you with consistent data handling as well as established and proven logic on data. This makes possible advanced real-time reporting of any kind.     
```

Comment:
```suggestion
- You can individually adapt the dynamic data models to the specific needs of your company. You can easily adjust data streams so that you do not only manage data, but also understand it. The platform provides you with consistent data handling as well as established and proven logic on data. This makes possible advanced real-time reporting of any kind.     
```